content/docs/dvclive/how-it-works.md

```diff
@@ -68,23 +70,67 @@ model.pt
 model.pt.dvc
 ```
 
-## Track the results
+## Tracking with Git+DVC
 
 DVCLive expects each run to be tracked by Git, so it will save each run to the
-same path and overwrite the results each time. If you
-[run experiments](/doc/user-guide/experiment-management/running-experiments)
-using DVC <abbr>pipelines</abbr>, DVC will track them for you so you don't have
-to Git commit each run. <abbr>DVC experiments</abbr> are Git commits that DVC
-can find but that don't clutter your Git history or create extra branches in
-your repo.
+same path and overwrite the results each time. Include
+[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to auto-track
+as a <abbr>DVC experiment</abbr>. DVC experiments are Git commits that DVC can
+find but that don't clutter your Git history or create extra branches.
 
-If you don't have a DVC pipeline, you can include
-[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to save the
-results as a DVC experiment.
+### Track large artifacts with DVC
 
-When using `Live.log_artifact("model.pt")`, DVCLive will
+Models and data are often large and aren't easily tracked in Git.
+`Live.log_artifact("model.pt")` will
 [cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
-to avoid tracking large artifacts in Git. It will generate a `model.pt.dvc`
-metadata file, which you should track in Git. You can
-[retrieve](/doc/start/data-management/data-versioning#retrieving) the artifact
-from the Git commit.
+and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
+can be tracked in Git and becomes part of the experiment. With this metadata
+file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
+the versioned artifact from the Git commit.
+
+### Run with DVC
+
+Experimenting in Python interactively (like in notebooks) is great for
+exploration, but eventually you may need a more structured way to run
+reproducible experiments (for example, running a parallelized hyperparameter
+search). By configuring DVC <abbr>pipelines</abbr>, you can
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+with `dvc exp run`.
+
+DVCLive prints instructions for how to start:
+
+```
+To run with DVC, add this to `/Users/sarah/myproject/dvc.yaml`:
```

Comment:
better move the line out, or make it a cooment and use `yaml` as a highlighter?

Reply:
> better move the line out

What does it mean? I was trying to show the instructions as they will be shown in the logger, but I could only show the YAML part or omit the actual output altogether (which would be easier for maintenance).

Reply:
sorry, I mean something like:

DVCLive prints instructions `To run with DVC, add this to "/Users/sarah/myproject/dvc.yaml":`

```yaml
stages:
  dvclive:
    cmd: <python my_code_file.py my_args>
    deps:
    - <my_code_file.py>
    outs:
    - dvclive/params.yaml:
 ....
```

Use `yaml` syntax highlighter.

This is minor, stylistic comment.

Reply:
@shcheklein Got it. Should be fixed now, thanks!